src/scopes/activityReport/index.js

```diff
@@ -112,6 +117,6 @@ export const topicToQuery = {
   },
 };
 
-export function activityReportsFiltersToScopes(filters) {
-  return createFiltersToScopes(filters, topicToQuery);
+export function activityReportsFiltersToScopes(filters, options, userId) {
+  return createFiltersToScopes(filters, topicToQuery, options, userId);
```

Comment:
If you keep userid in ```options``` instead of extracting it, you could continue to pass options through here.